README.md

```diff
@@ -8,6 +8,16 @@
 This plugin deals with the authentication and fetching of the JSON schema from a live Jenkins instance.
 It provides the base for validation of yaml files written to configure a Jenkins instance.
 
+## How to use
+
+1) Download and install the plugin from the VSCode Marketplace.
+2) Inside the Settings tab of VSCode find JCasc which will display two input blocks:
+    a) URL fileld for the Jenkins instance that you want to to extract the YAML Schema from.
+    b) User Token Field for if the Jenkins instance is protected.
+3) Once this is set you can use `Ctrl + Shift + P` to activate the extension.
```

Comment:
and do what exactly?

Reply:
Should I change it to `Once this is set you can use `Ctrl + Shift + P` to activate the extension and download the schema`

Reply:
don't you need to use a command? it's not just activating it right?

Reply:
You just have to do a `Ctrl + Shift + P` and select the extension that activates it, we can I guess use a shortcut to activate it but it might collide with other extensions hence the simplicity.

Reply:
and do what, ctl shift p just brings up a dialog

Reply:
I updated the text kindly have a look.